thunderscope.py

```diff
@@ -70,15 +70,15 @@
     # Control / Status.
     ("fe_control", 0,
         Subsignal("fe_en",       Pins("J21"), IOStandard("LVCMOS33")), # TPS7A9101/LDO & LM27761 Enable.
-        Subsignal("coupling",    Pins("H20 K19 H19 N18"), IOStandard("LVCMOS33")),
-        Subsignal("attenuation", Pins("G20 K18 J19 N19"), IOStandard("LVCMOS33")),
-        Subsignal("term",        Pins("H22 L19 L21 M18"), IOStandard("LVCMOS33"))
+        Subsignal("coupling",    Pins("N18 H19 K19 H20"), IOStandard("LVCMOS33")),
```

Comment:
It would be good to drop a comment here defining the order since it has been a source of confusion in the past when it got switched up

Reply:
And that goes for this whole commit I guess